[2024-10-09 09:00:04,645] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: mssql_dag.query_receipt 2024-10-09T09:00:03.571725+00:00 [queued]>
[2024-10-09 09:00:04,654] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: mssql_dag.query_receipt 2024-10-09T09:00:03.571725+00:00 [queued]>
[2024-10-09 09:00:04,655] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-10-09 09:00:04,655] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2024-10-09 09:00:04,655] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-10-09 09:00:04,667] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): query_receipt> on 2024-10-09T09:00:03.571725+00:00
[2024-10-09 09:00:04,671] {standard_task_runner.py:52} INFO - Started process 96210 to run task
[2024-10-09 09:00:04,674] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'mssql_dag', 'query_receipt', '2024-10-09T09:00:03.571725+00:00', '--job-id', '31', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/mssql_dag.py', '--cfg-path', '/tmp/tmptrk2j_t5', '--error-file', '/tmp/tmp_6fokl1f']
[2024-10-09 09:00:04,679] {standard_task_runner.py:77} INFO - Job 31: Subtask query_receipt
[2024-10-09 09:00:04,836] {logging_mixin.py:104} INFO - Running <TaskInstance: mssql_dag.query_receipt 2024-10-09T09:00:03.571725+00:00 [running]> on host da843a407010
[2024-10-09 09:00:04,887] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=mssql_dag
AIRFLOW_CTX_TASK_ID=query_receipt
AIRFLOW_CTX_EXECUTION_DATE=2024-10-09T09:00:03.571725+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-10-09T09:00:03.571725+00:00
[2024-10-09 09:00:05,955] {python.py:151} INFO - Done. Returned value was: None
[2024-10-09 09:00:05,962] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=mssql_dag, task_id=query_receipt, execution_date=20241009T090003, start_date=20241009T090004, end_date=20241009T090005
[2024-10-09 09:00:05,997] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-09 09:00:06,010] {local_task_job.py:149} INFO - Task exited with return code 0
